presto-kafka/src/main/java/io/prestosql/plugin/kafka/encoder/RowEncoder.java

```diff
@@ -15,7 +15,10 @@
 
 import io.prestosql.spi.block.Block;
 
+import java.io.Closeable;
+
 public interface RowEncoder
+        extends Closeable
```

Comment:
Extract making `RowEncoder` Closable to separate commit.

Reply:
Sorry for not being precise.
I meant extracting the interface chage to separate commit which is before one which add `Avro encoder`.

Then `AvroRowEncoder.close()` can be added together with rest of Avro encoder code.